CondFormats/SiPixelObjects/src/SiPixelFrameConverter.cc

```diff
@@ -38,7 +38,7 @@ PixelROC const * SiPixelFrameConverter::toRoc(int link, int roc) const {
     stm << "Map shows no fed="<<theFedId
         <<", link="<< link
         <<", roc="<< roc;
-    LogDebug("SiPixelFrameConverter") << stm.str();
+    edm::LogWarning("SiPixelFrameConverter") << stm.str();
```

Comment:
@veszpv how often is this warning issued actually?
I am just concerned that this might fill up logs at Tier0.

Reply:
It was changed to debug as a hack, I am putting the warning back now. Should never be issued after the PR. If so, it is a good reason to investigate.

Reply:
I haven't seen this show up in my test on run 300517 (LS \~600) on 4K events.

Reply:
It used to generate ~418 warning per event for uncabled FED channels.

Reply:
@veszpv, @slava77 thanks!